README.md

```diff
@@ -8,13 +8,202 @@
 + Delete a key-value pair from an object
 + Iterate over key-value pairs in an object
 
+## Objects in JavaScript
+
+What is an Object in JavaScript? An Object can formally be defined as a set
+of related variables, known as properties in this context, describing something
+specific. An Object can be anything you want it to be. To give a practical example,
```

Comment:
This first sentence is a little technobabble-heavy, but I really like the example that follows.

Do we want students to associate the kv pairs in Objects with variables? Could this be confusing for students? (Open question, I think.)